pkg/controller/volume/attachdetach/reconciler/reconciler.go

```diff
@@ -227,6 +235,29 @@ func (rc *reconciler) reconcile() {
 			// If timeout is true, skip verifySafeToDetach check
 			glog.V(5).Infof(attachedVolume.GenerateMsgDetailed("Starting attacherDetacher.DetachVolume", ""))
 			verifySafeToDetach := !timeout
+			if utilfeature.DefaultFeatureGate.Enabled(features.NodeShutdown) {
```

Comment:
@saad-ali - I was also thinking about doing this in the asw and dsw so I’m fine with this.